dropwizard-servlets/src/main/java/io/dropwizard/servlets/tasks/TaskServlet.java

```diff
@@ -91,6 +95,25 @@ public void add(Task task) {
     }
 
     @Override
+    protected void doGet(HttpServletRequest req,
+                         HttpServletResponse resp) throws ServletException, IOException {
+        if (Strings.isNullOrEmpty(req.getPathInfo())) {
+            try (final PrintWriter output = resp.getWriter()) {
+                resp.setContentType(MediaType.PLAIN_TEXT_UTF_8.toString());
+                final ArrayList<Task> tasks = new ArrayList<>(getTasks());
```

Comment:
Nice, thanks @joschi, nice and clean. (Grepping for `stream()` made me think it was a bad idea to use it :D )